testdata/plan_stringindex/main.tf

```diff
@@ -0,0 +1,236 @@
+terraform {
```

Comment:
This is a larger test `main.tf` taken from https://registry.coder.com/modules/coder/jetbrains-gateway

I prefer it being a bit complicated like this :+1: